leash.go

```diff
@@ -434,6 +434,9 @@ func sendToLibhoney(toBeSent chan event.Event, toBeResent chan event.Event,
 		}
 		// otherwise pick something up off the regular queue and send it
 		select {
+		case <-ctx.Done():
```

Comment:
I think I'd rather leave this one out. When the CTRL-C comes in, it will close the tailer which will in turn close lines and we'll hit the closed channel once all the in-flight events are sent. I think this fits better with the expectation set about picking up tailing from where we left off. There is also the 10sec timer to really kill it if the queue is full enough that it can't get them out in that time, so adding this only changes time to exit my a max of 10sec.

Reply:
Nod. I was iffy about the guarantees around resuming, but you're right and impatient debug cyen of the past should just shove it :)